src/components/screens/passphraseBackup/index.js

```diff
@@ -1,72 +1,77 @@
-import React from 'react';
-import connect from 'redux-connect-decorator';
+import React, { useState } from 'react';
 import { View, ScrollView, SafeAreaView } from 'react-native';
 import { translate } from 'react-i18next';
 import QRCode from 'react-native-qrcode-svg';
 import { themes, colors } from 'constants/styleGuide';
-import {
-  deviceWidth,
-  deviceHeight,
-  SCREEN_HEIGHTS,
-} from 'utilities/device';
+import { deviceWidth, deviceHeight, SCREEN_HEIGHTS } from 'utilities/device';
+import { PrimaryButton } from 'components/shared/toolBox/button';
 import PassphraseCopy from 'components/shared/passphraseCopy';
 import withTheme from 'components/shared/withTheme';
 import { P, A } from 'components/shared/toolBox/typography';
+import HeaderBackButton from 'components/navigation/headerBackButton';
 import getStyles from './styles';
 
 const isSmallScreen = deviceHeight() < SCREEN_HEIGHTS.MD;
 const qrCodeSize = deviceWidth() * (isSmallScreen ? 0.64 : 0.72);
 
-@connect(state => ({
-  passphrase: state.accounts.passphrase,
-}))
-class PassphraseBackup extends React.Component {
-  state = {
-    passphraseRevealed: false,
-  };
+const PassphraseBackup = ({
+  styles,
+  t,
+  theme,
+  navigation,
+  sharedData: data,
```

Comment:
shall we chose better naming here
```suggestion
  recoveryPhrase,
```